src/Entity.ts

```diff
@@ -56,8 +56,9 @@ export function isPrebuilt(entity: EntityBase) {
   return (entity.entityName === `builtin-${entity.entityType.toLowerCase()}`)
 }
 
-export const MAX_ENUM_VALUES = 5
-export const MAX_ENUM_VALUE_LENGTH = 10
+export const MAX_ENUM_VALUES = 5  // TODO(thpar) : Obsolete, delete after references cleaned up
```

Comment:
Why did this need to still be here?

Reply:
Backwards compatibility; I thought we could remove it once the other packages are building against the new release of Models.  Can remove if we're ok just breaking compatibility until the others are updated, LMK.

Reply:
As long as you don't update the ref in package.json on the other projects it's ok.  We frequently need to push things that require changes across projects

Reply:
Ok; removed